aiida/cmdline/params/options/__init__.py

```diff
@@ -451,7 +452,43 @@ def decorator(command):
 
 FREQUENCY = OverridableOption('-F', '--frequency', 'frequency', type=click.INT)
 
-VERBOSE = OverridableOption('-v', '--verbose', is_flag=True, default=False, help='Be more verbose in printing output.')
+VERBOSE = OverridableOption('-v', '--verbose', count=True, help='Print more verbose output.')
+
+
+def VERBOSITY(logger='cli', *names, **kwargs):  # pylint: disable=keyword-arg-before-vararg,invalid-name
+    '''A decorator that adds a `--verbosity, -v` option to the decorated
+    command.
+
+    Name can be configured through ``*names``. Keyword arguments are passed to
+    the underlying ``click.option`` decorator.
+    '''
+    from aiida.common.log import LOG_LEVELS
+    levels_str = ', '.join(LOG_LEVELS.keys())
+
+    if not names:
+        names = ['--verbosity', '-v']
+
+    kwargs.setdefault('default', 'INFO')
+    kwargs.setdefault('metavar', 'LVL')
+    kwargs.setdefault('expose_value', False)
+    kwargs.setdefault('help', 'One of {}.'.format(levels_str))
```

Comment:
```suggestion
    kwargs.setdefault('help', Control the verbosity of messages emitted by the command by setting the minimum log level')
```
And maybe it would then be useful to turn this option into `type=click.Choice` setting the values to the list of log levels such that they are automatically validated and added to the help string.